src/main/scala/com.snowplowanalytics.sauna/actors/Mediator.scala

```diff
@@ -52,11 +56,14 @@ class Mediator(saunaSettings: SaunaSettings) extends Actor {
    */
   val observers: List[ActorRef] =
     localObserversCreator(saunaSettings).map { case (name, props) => context.actorOf(props, name) } ++
-    s3ObserverCreator(saunaSettings).map { case (name, props) => context.actorOf(props, name) }
+      s3ObserverCreator(saunaSettings).map { case (name, props) => context.actorOf(props, name) } ++
+      kinesisObserverCreator(saunaSettings).map { case (name, props) => context.actorOf(props, name) }
 
   // Terminate application if none observer were configured
   // null is valid value when overriding observers in tests
-  if (observers != null && observers.isEmpty) { stop(Some("At least one observer must be configured")) }
+  if (observers != null && observers.isEmpty) {
```

Comment:
`null is valid value when overriding observers in tests`, as mentioned in the comment above. Removing this particular check breaks some integration tests.